src/api/lib/tasks/errbit.rake

```diff
@@ -0,0 +1,6 @@
+namespace :errbit do
+  desc 'Raise an exception, can be used to test erbit setups'
+  task :test do
+    raise RuntimeError.new("Runtime error exception to test error handling")
```

Comment:
Okay, so if we don't need this functionality anymore, which I hope;-), the PR looks good to me